docs/installation.rst

```diff
@@ -14,6 +14,8 @@ Prerequisites
 1. Linux x86_64
 2. `CUDA 11.8 <https://developer.nvidia.com/cuda-downloads>`__
 3. |driver link|_ supporting CUDA 11.8 or later.
+4. `cuDNN 8.6 <https://developer.nvidia.com/cudnn>`__ or later (this requirement will be removed in the future for non-fused-attention use cases).
```

Comment:
Is that a requirement because we include cudnn frontend?

Reply:
No, it's because we included cudnn.h and checked find_package(cuDNN) in the cmake file as well. I intend to make this optional in the future.

Reply:
On a second thought, how do we make this optional? at compile time, we don't know if users will want to use FP8 fused attention or not.

Reply:
Well, including cudnn.h shouldn't make us require version 8.6, so I assumed that there is something else that specifically needs this version or later. I think it is fine to require cuDNN in general, but if we could make this require as old version as possible to not become a burden, that would be great.

Reply:
I found 8.6 because it's the lowest in all the NGC containers that come with CUDA 11.8. But I think we can take any cuDNN version, as long as it provides a cudnn.h and can pass the FindCUDNN.cmake. All the code that uses cuDNN is guarded by `#if CUDNN_VERSION >= 8900`, so it won't compile if the version is lower, but it won't cause any problems either. (Just error out if user wants to run FP8)

Reply:
I don't want to say `any` cuDNN version because I don't know if any version will have these libraries that the FindCUDNN.cmake will look for.
    CUDNN::cudnn_adv_train
    CUDNN::cudnn_ops_train
    CUDNN::cudnn_cnn_train
    CUDNN::cudnn_adv_infer
    CUDNN::cudnn_cnn_infer
    CUDNN::cudnn_ops_infer

Reply:
Replaced with cuDNN 8 as the requirement, per our discussion.